src/dsp_tools/analyse_xml_data/construct_and_analyze_graph.py

```diff
@@ -112,12 +113,12 @@ def make_graph(
     node_indices = list(graph.add_nodes_from(nodes))
     id_to_rustworkx_index = dict(zip(all_resource_ids, node_indices))
     rustworkx_index_to_id = dict(zip(node_indices, all_resource_ids))
```

Comment:
again, please remove "rustworkx" from variables.

Also, I would strongly encourage you to use shorter variable names, if the variables only live for 10 lines of code. There is no need for them to be so long, and it will make the code far more readable.  
I would suggest `index_lookup` and `id_lookup`

Reply:
done